psalm-baseline.xml

```diff
@@ -436,6 +364,20 @@
     <NullArgument occurrences="1">
       <code>null</code>
     </NullArgument>
+    <PossiblyUndefinedArrayOffset occurrences="1">
+      <code>$inputSpec['validators']</code>
+    </PossiblyUndefinedArrayOffset>
+  </file>
+  <file src="test/Element/CheckboxTest.php">
+    <PossiblyInvalidMethodCall occurrences="1">
+      <code>getHaystack</code>
+    </PossiblyInvalidMethodCall>
+    <PossiblyUndefinedArrayOffset occurrences="1">
+      <code>$inputSpec['validators']</code>
+    </PossiblyUndefinedArrayOffset>
+    <UndefinedInterfaceMethod occurrences="1">
+      <code>getHaystack</code>
+    </UndefinedInterfaceMethod>
```

Comment:
Inference regressed here: can we investigate why?

Reply:
This is just as I was saying above (afaik) that this is what is being tested for. So I'm not sure why these are showing now and not before (my theory: better types in the docblocks in src).